.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+name: Dependabot Dependency Submission
+
+on:
+  push:
+    branches: [ main ]
+
```

Comment:
question: we're not running this during pull requests like snyk, is this intentional?